examples/big_bank.py

```diff
@@ -152,14 +152,16 @@ def create_big_bank_workspace():
     single_page_application.tags.add(web_browser_tag)
     mobile_app = internet_banking_system.add_container(
         "Mobile App",
-        "Provides a limited subset of the Internet banking functionality to customers via their mobile device.",
+        "Provides a limited subset of the Internet banking functionality to "
+        + "customers via their mobile device.",
```

Comment:
As this is within parentheses, string concatenation should happen automatically.

```suggestion
        "customers via their mobile device.",
```